torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -308,7 +308,11 @@ void Reducer::mark_variable_ready_dense(VariableIndex index) {
                       "bucket_view.sizes() = ", bucket_view.sizes(),
                       "bucket_view.strides() = ", bucket_view.strides());
     }
-    bucket_view.copy_(grad, /* non_blocking */ true);
+    // imitates wrapped_scalar_tensor in ATen/native/BinaryOps.cpp
+    auto wrapped = c10::scalar_to_tensor(float(1.0/process_group_->getSize()));
+    wrapped.unsafeGetTensorImpl()->set_wrapped_number(true);
+    // Divides while copying into the bucket view.
+    at::native::mul_out(bucket_view, grad, wrapped);
```

Comment:
This fuse op LGTM!

Reply:
oops, forgot to add an equivalent in [mark_variable_ready_sparse](https://github.com/pytorch/pytorch/pull/34904/commits/ca160e4841850b0ca77dea7a817360a3e7c861c0#diff-c269190a925a4b0df49eda8a8f6c5bd3R340).